context/app/static/js/components/detailPage/BulkDataTransfer/usePanelSet.jsx

```diff
@@ -14,6 +15,8 @@ const dbGaPText = {
     'The database of Genotypes and Phenotypes archives and distributes data and results from studies that have investigated the interaction of genotype and phenotype in humans.',
 };
 
+const dbGaPTooltip = <InfoTooltipIcon iconTooltipText="Database of Genotypes and Phenotypes" noMargin />;
```

Comment:
Should we just add the `dbGaP` text here to avoid repeating it below?

Reply:
I'm not sure where the repetition you're referring to is occurring, to be honest - I believe you may be referring to the slightly different cases for `dbGapPanel` and `noGlobusAccessWhileLoggedInPanel`?

Reply:
The `dbGaPTooltip` is generally preceded by the word `dbGaP`. Don't worry about it!